troubadix/plugins/description.py

```diff
@@ -38,7 +38,7 @@ def run(
             file_content: The content of the file that is going to be
                           checked
         """
-        pattern = re.compile(r"script_description\(.+\);", re.IGNORECASE)
+        pattern = re.compile(r"script_description", re.IGNORECASE)
```

Comment:
why?
What about `pattern = re.compile(r"script_description\s*\(\s*.+\s*\)\s*;", re.IGNORECASE)`
Or what about just check `if "script_description" in content` (make it a `FileContentPlugin`)?

Reply:
Thought it would be nice to have a line.
if "script_description" in content is better.